contracts/test/strategies/aerodrome-amo.base.fork-test.js

```diff
@@ -471,18 +475,16 @@ describe("ForkTest: Aerodrome AMO Strategy (Base)", async function () {
 
   describe("Harvest rewards", function () {
     it("Should be able to collect reward tokens", async () => {
-      const strategistAddr = await strategist.getAddress();
-
       await setERC20TokenBalance(
         aerodromeAmoStrategy.address,
         aero,
         "1337",
         hre
       );
-      const aeroBalanceBefore = await aero.balanceOf(strategistAddr);
-      await aerodromeAmoStrategy.connect(strategist).collectRewardTokens();
+      const aeroBalanceBefore = await aero.balanceOf(harvester.address);
```

Comment:
Thanks for fixing this, I believe I also fixed this in another branch: #2275 
Depending on which PR goes live first, we might have a merge conflict. But I can change my branch if it comes to that

Reply:
thanks for flagging. Yeah let this be a race and the last PR to be merged needs to fix it :)